pkg/task/restore.go

```diff
@@ -23,7 +23,8 @@ import (
 )
 
 const (
-	flagOnline = "online"
+	flagOnline        = "online"
+	flagSkipCreateSQL = "skip-create-sql"
```

Comment:
If we follow Lightning this flag should be named `--no-schema`

Reply:
change it to --no-schema